src/fontra/client/core/utils.js

```diff
@@ -546,9 +546,10 @@ export function areGuidelinesCompatible(parents) {
 
 const identityGuideline = { x: 0, y: 0, angle: 0 };
 
-export function normalizeGuidelines(guidelines) {
+export function normalizeGuidelines(guidelines, resetLocked = false) {
   return guidelines.map((guideline) => {
-    return { ...identityGuideline, ...guideline, locked: false };
+    const guidelineLocked = resetLocked ? false : guideline.locked ? true : false;
```

Comment:
```suggestion
    const guidelineLocked = resetLocked ? false : !!guideline.locked;
```
And then you might as well inline the expression in the line below again.